CONTRIBUTING.md

```diff
@@ -0,0 +1,325 @@
+```sh
+% git checkout -b <branch name>
+```
+##### 3.After moving to website/directory, run npm install to install the necessary packages
+```sh
+% cd wesite/
+% npm install
+```
+##### 4.Check the display on localhost 
+```sh
+% npm run start
+```
+
+#### Adding Contents
+
+##### Adding docs page
+If you want to add a reference page for a component, follow this guide to add.
+###### 1.Create a new markdown file in the docs/ directory
+Ex) docs/components/Button.md
+```sh
+---
+id: button
+title: Button
+---
+
+My new content here..
+```
+###### 2.Specify id of the newly added page in website/sidebars.json
+```sh
+// Add button to the Getting Started category of docs
+{
+  "docs": {
+    "components": [
+      "attachment",
+      "button" // new doc here
+    ],
+    ...
+  },
+  ...
+}
+```
+##### Adding website/blog page
+If you want to add blog articles, follow this guide to add.  
+###### 1.Verify that there is a Link  to Blog in headerLinks of website/siteConfig.js
+```sh
+headerLinks: [
+    ...
+    { blog: true, label: 'Releases' },
+    ...
+]
+```
+###### 2.Create a new file in website/blog with this naming rule: YYYY-MM-DD-{Releases title}.md
+Ex) website/blog/2017-12-14-introducing-docusaurus.md
+```sh
+---
+author: Mikei
+authorURL: https://twitter.com/mikei
+authorFBID: 503283835
+title: New Blog Post
+---
+
+My new content here...
+```
+##### Adding items for Top navigation bar
+###### 1.Specify external links, custom pages, documents that you want to add in the headerLinks of website/siteConfig.js
+```sh
+{
+  headerLinks: [
+    ...
+    /* you can add docs */
+    { doc: 'my-examples', label: 'Examples' },
+    /* you can add custom pages */
+    { page: 'help', label: 'Help' },
+    /* you can add external links */
+    { href: 'https://github.com/facebook/docusaurus', label: 'GitHub' },
+    ...
+  ],
+  ...
+}
+```
+##### Adding custom page
+docusaurus uses React components to build a page.  
+components are saved as JS files in website/pages/en  
+###### 1.Create a page in website/pages/en
+###### 2.Specify this page in the headerLinks of website/siteConfig.js If you want to link it to the top navigation bar.
+```sh
+{
+  headerLinks: [
+    ...
+    { page: 'my-new-custom-page', label: 'My New Custom Page' },
+    ...
+  ],
+  ...
+}
+```
+#### Build
+##### 1.Move to  website/directory, run npm run build to build
+```sh
+cd website/
+npm run build
+```
+##### 2. Verify that a build/directory is created and it contains the .html of other pages and all the documents in the website/ directory
+
+#### Versioning
+##### 1.After moving to the website/ directory, specify the version’s number to generate the document of this corresponding version.
+```sh
+cd website/
+npm run version <version number>
+```
+##### 2.Verify that website/versioned_docs/version-{version number} and website/versioned_sidebars/version-{version number}-sidebars.jsonare created.
+##### 3.Verify the specified version is added in website/versions.json
```

Comment:
This section is too long, so I think we can add a link to this. (maybe need to update the target content though)
https://github.com/kintone-labs/kintone-ui-component/tree/master/docs/document